features/issues/components/issue-filters/use-debounce-value.ts

```diff
@@ -0,0 +1,18 @@
+import { useState, useEffect } from "react";
+
+export function useDebounceValue(inputValue: string, delay = 1000) {
```

Comment:
Nice implementation. Alternatively you could look at the `lodash.debounce` function